wpa_supplicant/wpa_gui-haiku/WirelessConfigDialog.cpp

```diff
@@ -65,18 +66,17 @@ class WirelessConfigView : public BView {
 
 		int32 row = 0;
 		layout->AddItem(fNetworkName->CreateLabelLayoutItem(), 0, row);
-		layout->AddItem(fNetworkName->CreateTextViewLayoutItem(), 1, row++);
+		layout->AddItem(fNetworkName->CreateTextViewLayoutItem(), 1, row++, 2, 1);
 
 		BPopUpMenu* authMenu = new(std::nothrow) BPopUpMenu("authMode");
 		if (authMenu == NULL)
 			return;
 
-		fAuthOpen = new(std::nothrow) BMenuItem(
-			B_TRANSLATE_COMMENT("Open", "Open network"), NULL);
+		fAuthOpen = new(std::nothrow) BMenuItem(B_TRANSLATE("Open ‒ no encryption"), NULL);
 		authMenu->AddItem(fAuthOpen);
-		fAuthWEP = new(std::nothrow) BMenuItem(B_TRANSLATE("WEP"), NULL);
+		fAuthWEP = new(std::nothrow) BMenuItem(B_TRANSLATE("WEP ‒ insecure encryption"), NULL);
 		authMenu->AddItem(fAuthWEP);
-		fAuthWPA = new(std::nothrow) BMenuItem(B_TRANSLATE("WPA/WPA2"), NULL);
+		fAuthWPA = new(std::nothrow) BMenuItem(B_TRANSLATE("WPA/WPA2 ‒ good encryption"), NULL);
```

Comment:
Well, it's nicely symmetrical over all options - no/insecure/good encryption. And when we support WPA3 we can add a "best encryption". :)

Reply:
But there's also WPA2 Enterprise, or WPA2 modes where everyone has their own access key, etc. So it's not really "best" exactly because there's yet more options under that. I get that the symmetry is nice, but I think we should aim for precision here.

Reply:
Fair enough. I updated the PR accordingly.